cli/jobs/single-step/pytorch/cifar-distributed/job.yml

```diff
@@ -10,7 +10,8 @@ inputs:
   learning_rate: 0.2
   cifar: 
      type: uri_folder
-     path: azureml:cifar-10-example:3
+     path: azureml:cifar-10-example@latest
+     mode: ro_mount
```

Comment:
default is ro_mount too, so its not needed to add mode setting fyi

Reply:
I have removed the mode parameter